### DISCUSSION TITLE
The LangchainCallbackHandler supports statistical delay of the first token

### DISCUSSION THREAD

**Discussion by Hacky-DH** [langfuse/langfuse#3129] [Ideas]
### Describe the feature or potential improvement

In the on_llm_new_token method of the LangchainCallbackHandler class, update the completion_start_time parameter of the generation using the current time. This way, you will be able to see the value of Time to First Token on the front-end page.

**Comment by hassiebp** [comment]
Thanks for raising this, indeed a good improvement!

**Reply by marcklingen** [reply]
tracking this here: https://github.com/langfuse/langfuse/issues/3432

**Comment by hassiebp** [comment]
@Hacky-DH Thanks again for your report, this feature has been now released 👍🏾 https://github.com/langfuse/langfuse-python/releases/tag/v2.51.0
